kolibri/core/device/api.py

```diff
@@ -103,6 +105,9 @@ def get(self, request, format=None):
         # Returns the named timezone for the server (the time above only includes the offset)
         info["server_timezone"] = settings.TIME_ZONE
         info["installer"] = installation_type()
+        info["python_version"] = "{major}.{minor}".format(
+            major=version_info.major, minor=version_info.minor
+        )
```

Comment:
I think we should have the entire contents of version_info here.

Maybe like this?

``` 
>>> from sys import version_info
>>> print(".".join(map(str, version_info)[:3]))
2.7.17
```

Reply:
Cool, thanks, I didn't know the micro part could be useful. I have added it (not using map, I prefered a more explicit  expression, but with the same result)

Reply:
Done, with explicit info

Reply:
I wasn't sure of `micro` would be a persistent property, so I used `map(str, version_info)[:3]`, but it might not be so?

Reply:
Well, we can't be sure `version_info` will be persistent either, if the Python API (https://docs.python.org/3/library/sys.html#sys.version_info)  changes we are done for sure :)
I prefer to follow the Zen of Python:
```
import this
...
Explicit is better than implicit.
Simple is better than complex.
...
```

Reply:
Thanks, I hadn't looked it up, it's because I know these components as major, minor and patch. I never heard "micro" before :)